src/table/table.js

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, Fragment } from "react";
+import PropTypes from "prop-types";
+import BaseSlideExecution from "../base-slide-execution";
+import { getFirstMediaUrlFromField, ThemeStyles } from "../slide-util";
+import "./table.scss";
+
+/**
+ * Table component.
+ *
+ * @param {object} props Props.
+ * @param {object} props.slide The slide.
+ * @param {object} props.content The slide content.
+ * @param {boolean} props.run Whether or not the slide should start running.
+ * @param {Function} props.slideDone Function to invoke when the slide is done playing.
+ * @returns {object} The component.
+ */
+function Table({ slide, content, run, slideDone }) {
+  // Styling from content
+  const { fontSize, fontPlacement } = content.styling || {};
+  const textClasses = `text ${fontSize}`;
+
+  // Content
+  const { table, title, text } = content;
+  const header = table.shift();
```

Comment:
This is a risky assumption. Do we know that the first element is always a header row?